components/automate-ui/src/app/components/notifications/notifications.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component } from '@angular/core';
+import { Observable } from 'rxjs';
+import { Store } from '@ngrx/store';
+import { NgrxStateAtom } from 'app/ngrx.reducers';
+
+import { notificationState } from 'app/entities/notifications/notification.selectors';
+import { Notification } from 'app/entities/notifications/notification.model';
+import { DeleteNotification } from 'app/entities/notifications/notification.actions';
+
+@Component({
+  selector: 'chef-notifications',
+  templateUrl: './notifications.component.html',
+  styleUrls: ['./notifications.component.scss']
+})
+export class ChefNotificationsComponent {
```

Comment:
Same thought as above: suggest delete the Chef prefix on this.
Also, that more closely follows convention of the file name matching the component name. 😁